src/P5Functions/Math.js

```diff
@@ -433,7 +433,14 @@ module.exports = function withMath(p, undef) {
       return internalRandomGenerator() * arguments[0];
     }
     var aMin = arguments[0], aMax = arguments[1];
-    return internalRandomGenerator() * (aMax - aMin) + aMin;
+    while (true) {
+      var ir = internalRandomGenerator();
+      var result = ir * (aMax - aMin) + aMin;
+      if (result !== aMax || result === aMin) {
```

Comment:
hmm, looking at https://github.com/processing-js/processing-js/issues/244#issuecomment-229767164 it feels like `random` might be behaving "as intended", even if that means it's not quite doing the right thing an sich, so I'm not sure about landing this PR now... did it fix a parity problem between how Processing works and how Processing.js runs the code for random?

(The basic guiding principle in Pjs is not "make it work properly", but "make it work the same as Processing", so if Processing breaks on things like random(10,-5) or random(2,2), then Pjs reasonably should, too)